template_src/package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name": "my_app",
+  "version": "1.0.0",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "none specified",
+  "license": "none specified"
```

Comment:
right. version defaults to 1.0.0. my bad.
should we get rid of package.json here? 
I made this basic starter package.json and there aren't dependencies in the original package.json